### ISSUE TITLE
Create environment for classic.fec.gov

### ISSUE THREAD

**Issue by PaulClark2** [18F/fec-transition#154]
So users can access classic.fec.gov after we shutdown our Waltham data center, spin-up web servers and application servers in alternative hosting center. 

- [ ] create web server in DC3
- [ ] create application server in DC3
- [ ] create read-only database in DC3
- [ ] copy classic.fec.gov web server content from Waltham web server to web server
- [ ] copy classic.fec.gov application server content from Waltham web server to application server
- [ ] test

**Comment by PaulClark2** [comment]
App server set up in AWS 
Some static pages available

No DB connection yet

**Comment by gmengistu** [comment]
Setting up website now. copying files over - minus audio files. Once that is done, will work on config. app maybe next week. Rohan is going to be working on DB over the weekend.

**Comment by PaulClark2** [comment]
@fec-jli  working on mod_jk

**Comment by rjayasekera** [comment]
I have created Webserver/Appserver (Apache/Tomcat/mod_jk) running at AWS 
connecting to a Database at DC4 (behind a firewall)
- Database is created and some data is uploaded. 
- There is not much space in it to copy the complete data set from BO1
- I assume we can drop real_efile data
- Still cannot connect to this DB from outside of FEC HQ
- Mark is working on opening up ports to be able to connect to DB
- Gashaw is working on adding more space and/or replace this server
- Current storage capacity is 100G
- Gashaw is also working on the official site (beside this prototype)
- I have copied some static files and style sheets from current web/app server
- AWS APP/WEB is a micro instance with 8GB storage and 1GB of memory
- Not a fully fictional website since no DB connections and not all files are loadeded
- Prototype / proof of concept works
- New site can be access from the following link (from any location): 
http://52.202.1.251/data/DataCatalog.do

**Comment by LindsayYoung** [comment]
Just wanted to drop a huge thank you to @rjayasekera and @gmengistu!

**Comment by rjayasekera** [comment]
DC4/DMZ database setup completed
	- Create DB
	- Export data from Waltham and import to DC4/DMZ
	
AWS webserver setup complete (Prototype)
	Most of the web resources copied from fec-web5

AWS appserver setup complete (Prototype)
	Most of the application war files copied from fec-app7
	
Most of the applications works.  Need to copy complete set of web resources 
and applications to a permanent web and app server in AWS and enabled security.  
Then everything should work.  
(I cannot copy them all since I don't have access to most of the resources)

Todo:
committee_summary, candidate_summary, committee_report_summary, and fecviewer
still does not fully functional.  May work after stop redirects, and setup to
download "xml" files from S3 bucket and include all of web resource files

To test:
http://52.202.1.251/data/
http://52.202.1.251/disclosure/
http://52.202.1.251/disclosurep/
http://52.202.1.251/disclosurehs/
http://52.202.1.251/disclosureie/
http://52.202.1.251/auditsearch/
http://52.202.1.251/MUR/
http://52.202.1.251/fecviewer/

**Comment by AmyKort** [comment]
@PaulClark2 Can you help update this ticket, or we can close it out and start another one? Thanks!

**Comment by PaulClark2** [comment]
Closing in favor of https://github.com/18F/fec-transition/issues/161
